src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/w3c/harvesters/JsElements.kt

```diff
@@ -0,0 +1,43 @@
+package com.atlassian.performance.tools.jiraactions.api.w3c.harvesters
+
+import com.atlassian.performance.tools.jiraactions.api.w3c.PerformanceElementTiming
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.TimeoutException
+
+
+internal fun getJsElementsPerformance(javascript: JavascriptExecutor): List<PerformanceElementTiming> {
+    return try {
+        val jsElements =
+            javascript.executeAsyncScript("new PerformanceObserver(list => arguments[0](list.getEntries())).observe({type: 'element', buffered: true})")
```

Comment:
What does the `buffered: true` do?

Reply:
You can attach JS listener after some time and still get measures that were registered/emitted by the browser before loading JS.